### PR TITLE
Revamp repo doc

### PR DESCRIPTION
**Pull request by harryskim**
Updated nGraph repo readme.md with new logo. Also made changes to few sentences and modified stack diagram to reflect the current support for FW & HW. 

Updated stack diagrams in "Archtecture & features" section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,44 +30,39 @@ The Python wheels for nGraph have been tested and are supported on the following
...
-developers. 
+nGraph Compiler aims to accelerate developing AI workloads using any deep learning
+framework and deploying to a variety of hardware targets. We strongly believe in 
+providing freedom, performance, and ease-of-use to AI developers. 
 
-The diagram below shows what deep learning frameworks and hardware targets
-we support. More details on these current and future plans are in the [ecosystem]
-section. 
+The diagram below shows deep learning frameworks and hardware targets
+supported by nGraph. NNP-L (Neural Network Processor for Learning) & NNP-I
```
Comment (indie) [inline README.md]: These should be referenced according to namesDB; see slack msg please

File: ABOUT.md [hunk 2]
```diff
@@ -117,7 +116,7 @@ release of nGraph. nGraph currently has limited support for dynamic graphs.
 Current nGraph Compiler full stack
 ----------------------------------
 
-![](doc/sphinx/source/graphics/about_fullstack.png)
```
Comment (postrational) [inline ABOUT.md]: Shouldn't you delete `about_fullstack.png`?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# nGraph Compiler Stack (Beta)
-
+![](doc/sphinx/source/graphics/ngraph_header.png)
```
Comment (postrational) [inline README.md]: Would be good to add alt text in the `[]` something like:
`nGraph Compiler Stack`

**Comment by harryskim** [comment]
All changes implemented.

**Pull request merged**